pxc-57/entrypoint.sh

```diff
@@ -130,21 +130,26 @@ set +e
 ipaddr=$(hostname -i | awk ' { print $1 } ')
 hostname=$(hostname)
 
-curl http://$DISCOVERY_SERVICE/v2/keys/pxc-cluster/queue/$CLUSTER_NAME -XPOST -d value=$ipaddr -d ttl=60
+#if ETCD_PROTOCOL is https and insecure curl is enabled, add -k to curl with CURL_OPTS
+if [ ${ETCD_PROTOCOL} == "true" ] && [ ${ETCD_CURL_INSECURE_SSL} == "true" ]; then
```

Comment:
according to previous comment
```suggestion
if [ ${ETCD_PROTOCOL} == "https" ] && [ ${ETCD_CURL_INSECURE_SSL} == "true" ]; then
```